modules/apiserversource-kn.adoc

```diff
@@ -0,0 +1,147 @@
+
+.Procedure
+
+. Create a service account, role and role binding for the ApiServerSource.
++
+You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default <1>
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: event-watcher
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - get
+      - list
+      - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: k8s-ra-event-watcher
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: event-watcher
+subjects:
+  - kind: ServiceAccount
+    name: events-sa
+    namespace: default <1>
+----
++
+<1> Adjust this namespace to the namespace that you have selected for installing ApiServerSource.
++
+[NOTE]
+====
+If you want to re-use an existing service account with the appropriate permissions, you must modify the `authentication.yaml` for that service account.
+====
++
+Create the service account, cluster role and cluster binding by entering the following command:
++
+----
+$ oc apply --filename authentication.yaml
+----
+
+. Create an ApiServerSource event source by entering the following command:
++
+----
+$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa --mode Resource
+----
+
+. To check that the ApiServerSource is set up correctly, create a Knative service that dumps incoming messages to its log by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display
+----
+
+. Create a trigger from the `default` broker to filter events to the service created in the previous step by entering the following `kn` command:
++
+----
+$ kn trigger create event-display-trigger --broker default --sink svc:event-display \ source=dev.knative.samples/testevents \
+----
+
+. Create events by launching a pod in the default namespace. You can do this by entering the following command:
++
+----
+$ oc create deployment hello-node --image=gcr.io/hello-minikube-zero-install/hello-node
+----
+
+.Verification steps
+
+You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+
+You can view the message dumper function logs by entering the following commands:
+----
+$ oc get pods
+$ oc logs $(oc get pod -o name | grep event-display) -c user-container
+----
+
+The logs should contain lines similar to the following:
+----
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.apiserver.resource.update
+  datacontenttype: application/json
+  ...
+Data,
+  {
+    "apiVersion": "v1",
+    "involvedObject": {
+      "apiVersion": "v1",
+      "fieldPath": "spec.containers{busybox}",
```

Comment:
If we do keep the "hello-node" example above, we should make this output consistent with it, so at least replace all occurences of "busybox" with "hello-node"

Reply:
Can we first confirm if that example is definitely OK to use? I don't want to update this to update it again later ;)